pyproject.toml

```diff
@@ -9,7 +9,11 @@ name = "halmos"
 description = "Halmos: Symbolic Bounded Model Checker for Ethereum Smart Contracts Bytecode"
 readme = "README.md"
 authors = [
+    { name="a16z crypto" },
+]
+maintainers = [
     { name="Daejun Park" },
+    { name="karmacoma" },
```

Comment:
Karma, please feel free to update your name, and/or include your email if you'd like.